File: FrontEnd/GameProject.UI/src/app/shared/models/bussiness/dtos/change-game-status-request.ts
import {GameStatus} from "../../game-status";
import {GameAllInfo} from "../../rawg-api/games/game-all-info";

export interface ChangeGameStatusRequest {
  oldStatus: GameStatus;
  newStatus: GameStatus;
  game: GameAllInfo;
  gameRawgId: number;
  platform: string;
}

File: BackEnd/GameProject.Application/Models/Bussiness/Requests/ChangeGameStatusRequest.cs
using System.ComponentModel.DataAnnotations;
using GameProject.Application.Models.Shared;
using GameProject.Domain.Models;

namespace GameProject.Application.Models.Bussiness;

public class ChangeGameStatusRequest : BaseRequest
{
    public GameStatus OldStatus { get; set; }
    public GameStatus NewStatus { get; set; }
    public GameAllInfo Game { get; set; }
    public int GameRawgId { get; set; }
    
    [Required]
    public string Platform { get; set; } 
}